plugins/module_utils/vardict.py

```diff
@@ -93,7 +106,7 @@ def __str__(self):
 
 
 class VarDict(object):
-    reserved_names = ('__vars__', 'var', 'set_meta', 'set', 'output', 'diff', 'facts', 'has_changed')
+    reserved_names = ('__vars__', '_var', 'var', 'set_meta', 'get_meta', 'set', 'output', 'diff', 'facts', 'has_changed', 'as_dict')
```

Comment:
Hmm, isn't changing this field a breaking change (since now using `_var`, `get_meta`, and `as_dict` is illegal and will result in an exception)?

Reply:
I guess it is - I had not considered it from that perspective. Will update the changelog fragment, but cannot escape from that.

Reply:
done

Reply:
Hmm, in that case we should probably only merge it for 8.0.0 and not backport it to stable-7.

Reply:
makes sense to me